src/emulator/constants.ts

```diff
@@ -3,6 +3,7 @@ import * as url from "url";
 import { Address, Emulators } from "./types";
 
 const DEFAULT_PORTS: { [s in Emulators]: number } = {
+  gui: 3000,
```

Comment:
This is an extremely common port for dev tools that I have used in the past, can we choose something else?  I know we hunt within 100 ports but it would be nice to avoid constant warnings.

Reply:
Changed to 4000+.